exec/cnex/dictionary.c

```diff
@@ -96,8 +133,21 @@ void dictionary_freeDictionary(Dictionary *self)
     free(self);
 }
 
+static  int dictionary_compareKeys(const Cell *a, const Cell *b)
```

Comment:
To avoid warnings, this should be declared as `qsort()` expects: `(const void *, const void *)`. Then, inside this function, cast the two pointers to `const Cell *`.

Reply:
Fixed.